public/index.html

```diff
@@ -1,6 +1,12 @@
 <!doctype html>
 <html>
-  <head></head>
+  <head>
+    <style>
+      html {
+        font-size: 62.5%;
+      }
+    </style>
+  </head>
```

Comment:
Studio has this style, so I'm replicating it here. Without this style, the fonts in bare studio-frontend would be ginormous because all of the rem units are assuming that this is the base font size.